content/tutorials/history.textile

```diff
@@ -33,6 +33,56 @@ You have now enabled history for all channels in the @persisted@ namespace i.e.
 
 h2. Step 3 - Install Ably
 
+blang[java].
+  To start using Ably in your Android app, you need to include the Ably Client library. We recommend that you include the latest client library via Gradle in your module-level @gradle.build@ file.
+
+  ```[groovy]
+  apply plugin: 'com.android.application'
+  ...
+  dependencies {
+      ...
+      compile 'io.ably:ably-java:0.8.3'
+  }
+  ```
+
+  Also add following repo for a sub-dependency in your project-level @gradle.build@ file.
+  
+  ```[groovy]
+  allprojects {
+    repositories {
+        ...
+        maven { url "https://raw.github.com/paddybyers/Java-WebSocket/mvn-repo/" }
+    }
+  }
+  ```
+  
+  After you add neccessary dependencies, you can import AblyRest class into your code and initialize it.
+
+  ```[java]
+  import io.ably.lib.rest.AblyRest;
+
+  public class ExampleActivity extends AppCompatActivity {
+    private final static String API_KEY = "INSERT-YOU-API-KEY-HERE"; 
+    ...
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_example);
+        try {
+            initAbly();
+        } catch (AblyException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void initAbly() throws AblyException {
+        AblyRest ablyRest = new AblyRest(API_KEY);
+    }
+  }
+  ```
+
+  "See this step in Github":https://github.com/piotrekinventica/tutorials/commit/4619056
```

Comment:
Unfortunately at this point I have not been given instructions on how to get the dependencies or how to run this demo?  I think we have to assume that people are not necessary familiar with any of the tools we use so we must be explicit on what steps a user must take to get this to work.